### DISCUSSION TITLE
Adding "List of Abbreviations" to the Thesis

### DISCUSSION THREAD

**Discussion by nhoelzl** [Digital-Media/HagenbergThesis#130] [Q&A] [answered]
Is there a way to add abbreviations smoothly? I have troubles using ` \usepackage{acro}`.  My  "List of Abbreviations" does not show up in Table of Contents.

**Comment by hochleitner** [comment, marked as answer]
Did you follow the [guide in the wiki](https://github.com/Digital-Media/HagenbergThesis/wiki/List-of-Acronyms)? Did you include the `\printacronyms` command in the `backmatter`? If yes, are there any errors?

**Reply by nhoelzl** [reply]
Thank you for directing me to the wiki. I was not aware of this section. Thank you very much! It helped.

**Reply by nhoelzl** [reply]
I got an error when adding `\acsetup{subsequent-style=short}`.  
Error: LaTeX3 Error: The key 'acro/subsequent-style' is unknown and is being ignored.

**Reply by nhoelzl** [reply]
All the other definitions for `\acsetup` work.

**Reply by nhoelzl** [reply]
I found the answer [here](https://ctan.math.washington.edu/tex-archive/macros/latex/contrib/acro/acro-manual.pdf), on page 11.

According to documentation, the `\ac`  prints the acronym long-short the first use, then every subsequent use only in the abbreviated form.

**Reply by hochleitner** [reply]
> I got an error when adding `\acsetup{subsequent-style=short}`.
> Error: LaTeX3 Error: The key 'acro/subsequent-style' is unknown and is being ignored.

I suppose you're using a too old version of the `acro` package. According to the [acro manual](https://mirror.kumi.systems/ctan/macros/latex/contrib/acro/acro-manual.pdf), `subsequent-style` was introduced in version v3.4 (2020/12/25). You might have an older one. If you're using a local LaTeX installation, try running a package update.

If you're using Overleaf, then it's not quite so simple. Overleaf still runs TeX Live 2020, which has `acro` v.3.2. I tried uploading a current `acro.sty` (tried v3.4 and the current v3.5), but those obviously contain commands that create even more errors on Overleaf. Obviously, they rely on other packages that need to be updated, and Overleaf's TeX distro doesn't have them.

I suggest removing the `subsequent-style=short` rule, and you should be fine anyways.

**Reply by nhoelzl** [reply]
Oh yes, in my case I use Overleaf.
It works now for me with removing the `subsequent-style=short`. Thank you for explaining this to me. I was not aware that Overleaf relies on an older version of this.  :)
